subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/maven/MavenDependencyResolveIntegrationTest.groovy

```diff
@@ -209,4 +209,31 @@ task check << {
         expect:
         succeeds "check"
     }
+
+    def "dependency with wildcard exclusions is treated as non-transitive"() {
```

Comment:
I'm surprised we don't already have specific coverage for this. This PR doesn't touch resolution behavior so this should have been the case already.